loadtest/main.go

```diff
@@ -1,226 +1,226 @@
-					Price:             sdk.NewDec(int64(shortPrice)).Quo(FROM_MILI),
-					Quantity:          sdk.NewDec(int64(shortQuantity)).Quo(FROM_MILI),
-					PriceDenom:        "usdc",
-					AssetDenom:        "sei",
-					PositionEffect:    dextypes.PositionEffect_OPEN,
-					OrderType:         dextypes.OrderType_LIMIT,
-					Leverage:          sdk.NewDec(1),
-				})
-			}
-			amount, err := sdk.ParseCoinsNormalized(fmt.Sprintf("%d%s", longPrice*longQuantity+shortPrice*shortQuantity, "usei"))
-			if err != nil {
-				panic(err)
-			}
-			msg := dextypes.MsgPlaceOrders{
-				Creator:      sdk.AccAddress(key.PubKey().Address()).String(),
-				Orders:       orderPlacements,
-				ContractAddr: contractAddress,
-				Funds:        amount,
-			}
-			txBuilder := TEST_CONFIG.TxConfig.NewTxBuilder()
-			_ = txBuilder.SetMsgs(&msg)
-			seqDelta := uint64(i / 2)
-			SignTx(&txBuilder, key, seqDelta)
-			mode := typestx.BroadcastMode_BROADCAST_MODE_SYNC
-			if j == len(activeAccounts)-1 {
-				mode = typestx.BroadcastMode_BROADCAST_MODE_BLOCK
-			}
-			sender := SendTx(key, &txBuilder, mode, seqDelta, &mu)
-			wg.Add(1)
-			senders = append(senders, func() {
-				defer wg.Done()
-				sender()
-			})
-		}
-		sendersList = append(sendersList, senders)
-
-		tmp := inactiveAccounts
-		inactiveAccounts = activeAccounts
-		activeAccounts = tmp
-	}
-
-	lastHeight := getLastHeight()
-	for i := 0; i < int(numberOfBlocks); i++ {
-		newHeight := getLastHeight()
-		for newHeight == lastHeight {
-			time.Sleep(50 * time.Millisecond)
-			newHeight = getLastHeight()
-		}
-		fmt.Println(fmt.Sprintf("Sending %d-th block", i))
-
-		senders := sendersList[i]
-		wg := wgs[i]
-
-		for _, sender := range senders {
-			go sender()
-		}
-		wg.Wait()
-	}
-}
-
-func getLastHeight() int {
-	out, err := exec.Command("curl", "http://localhost:26657/blockchain").Output()
-	if err != nil {
-		panic(err)
-	}
-	var dat map[string]interface{}
-	if err := json.Unmarshal(out, &dat); err != nil {
-		panic(err)
-	}
-	result := dat["result"].(map[string]interface{})
-	height, err := strconv.Atoi(result["last_height"].(string))
-	if err != nil {
-		panic(err)
-	}
-	return height
-}
-
-func main() {
-	args := os.Args[1:]
-	contractAddress := args[0]
-	numberOfAccounts, _ := strconv.ParseUint(args[1], 10, 64)
-	numberOfBlocks, _ := strconv.ParseUint(args[2], 10, 64)
-	longPriceFloor, _ := strconv.ParseUint(args[3], 10, 64)
-	longPriceCeiling, _ := strconv.ParseUint(args[4], 10, 64)
-	shortPriceFloor, _ := strconv.ParseUint(args[5], 10, 64)
-	shortPriceCeiling, _ := strconv.ParseUint(args[6], 10, 64)
-	quantityFloor, _ := strconv.ParseUint(args[7], 10, 64)
-	quantityCeiling, _ := strconv.ParseUint(args[8], 10, 64)
-	chainId := args[9]
-	CHAIN_ID = chainId
-	run(
-		contractAddress,
-		numberOfAccounts,
-		numberOfBlocks,
-		longPriceFloor,
-		longPriceCeiling,
-		shortPriceFloor,
-		shortPriceCeiling,
-		quantityFloor,
-		quantityCeiling,
-	)
-}
+// import (
```

Comment:
this was commented to unblock linter CI for this PR, since the loadtest files aren't updated to match the new data structures. I plan to do that in a followup PR to not bloat up this PR too much